page.html

```diff
@@ -11,6 +11,9 @@
   <!---Added chosen plugin for dropdown-->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/chosen/1.4.2/chosen.css">
   <link rel="stylesheet" href="./assets/css/style.css">
+  <!--Mapbox Api-->
+  <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.js'></script>
```

Comment:
I would suggest moving this script tag to the bottom of the page with the other script tags, otherwise it looks great!